iterators_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,18 @@
 #include "impl.h"
 
+std::string reverseWords(const std::string& input)
+{
+    std::stringstream stream{input};
+
+    std::vector<std::string> parsed{std::istream_iterator<std::string>(stream),
+                std::istream_iterator<std::string>()};
+
+    stream = std::stringstream();
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es26-dont-use-a-variable-for-two-unrelated-purposes

Reply:
Fixed